src/mono/mono/metadata/loader.c

```diff
@@ -630,6 +605,190 @@ find_method (MonoClass *in_class, MonoClass *ic, const char* name, MonoMethodSig
 	return result;
 }
 
+static MonoMethod *
+find_method_simple (MonoClass *klass, const char *name, const char *qname, const char *fqname,
+		      MonoMethodSignature *sig, MonoClass *from_class, gboolean ignore_cmods, MonoError *error)
+{
+	MonoMethod *method_maybe = NULL;
+
+	/* FIXME: method refs from metadata-upate probably end up here */
```

Comment:
delete this comment. metadata-update will get handled by `find_method_slow`

Reply:
Removed.